synapse/handlers/room_member.py

```diff
@@ -1736,8 +1736,8 @@ async def _generate_local_out_of_band_leave(
             txn_id=txn_id,
             prev_event_ids=prev_event_ids,
             auth_event_ids=auth_event_ids,
+            outlier=True,
```

Comment:
For my own reference, by passing this here:

1. The event still gets it set on `internal_metadata` via `create_new_client_event`
2. Avoids calling `compute_event_context`, which (I think) is what creates the state group.

Reply:
correct!